files/default/sql/ddl/3.2.0__initial_tables.sql

```diff
@@ -1995,21 +1995,29 @@ CREATE TABLE IF NOT EXISTS `job_alert` (
                                            CONSTRAINT `fk_job_alert_2` FOREIGN KEY (`job_id`) REFERENCES `jobs` (`id`) ON DELETE CASCADE
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
 
-CREATE TABLE IF NOT EXISTS `feature_group_alert` (
-                                                     `id` int(11) NOT NULL AUTO_INCREMENT,
-                                                     `feature_group_id` int(11) NOT NULL,
-                                                     `status` varchar(45) COLLATE latin1_general_cs NOT NULL,
-                                                     `type` varchar(45) COLLATE latin1_general_cs NOT NULL,
-                                                     `severity` varchar(45) COLLATE latin1_general_cs NOT NULL,
-                                                     `receiver` int(11) NOT NULL,
-                                                     `created` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-                                                     PRIMARY KEY (`id`),
-                                                     UNIQUE KEY `unique_feature_group_alert` (`feature_group_id`,`status`),
-                                                     KEY `fk_feature_group_alert_2_idx` (`feature_group_id`),
-                                                     KEY `fk_feature_group_alert_1_idx` (`receiver`),
-                                                     CONSTRAINT `fk_feature_group_alert_1` FOREIGN KEY (`receiver`) REFERENCES `alert_receiver` (`id`) ON DELETE CASCADE,
-                                                     CONSTRAINT `fk_feature_group_alert_2` FOREIGN KEY (`feature_group_id`) REFERENCES `feature_group` (`id`) ON DELETE CASCADE
-) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+CREATE TABLE IF NOT EXISTS `hopsworks`.`feature_group_alert` (
+                                                             `id` int AUTO_INCREMENT
+                                                                 PRIMARY KEY,
+                                                             `feature_group_id` int NULL,
+                                                             `status` varchar(45) NOT NULL,
+                                                             `type` varchar(45) NOT NULL,
+                                                             `severity` varchar(45) NOT NULL,
+                                                             `receiver` int NOT NULL,
+                                                             `created` timestamp DEFAULT CURRENT_TIMESTAMP NOT NULL,
+                                                             `feature_view_id` int NULL,
+                                                             `entity_type` int NOT NULL,
```

Comment:
If entity_type can only have two values (i.e., 0 for feature group and 1 for feature view), then it's better to use `TINYINT(1) NOT NULL`